packages/studio/src/schemas/page.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from "zod";
+
+export const getEditPageSchema = z.object({
+  pageId: z.string(),
+  siteId: z.string()
+})
+
+export const updatePageSchema = z.object({
+  // NOTE: We allow both to be empty now, 
+  // in which case this is a no-op. 
+  // We are ok w/ this because it doesn't 
+  // incur any db writes
+  parentId: z.string().nullable(),
+  pageName: z.string().nullable(),
+  siteId: z.string(),
+  pageId: z.string()
+})
+
+export const updatePageBlobSchema = z.object({
```

Comment:
hmm actually some of this schema can be extended from one another! but think we can improver this later